misc/errors.go

```diff
@@ -2,57 +2,90 @@ package misc
 
 import (
 	"fmt"
-
-	"golang.org/x/xerrors"
+	"runtime"
+	"strings"
 )
 
+// wrapError 结构体用于包装错误信息
 type wrapError struct {
```

Comment:
在 `misc/errors.go` 中，`wrapError` 结构体的字段已经有注释，但建议统一注释格式。例如，使用 `// message 错误消息` 而不是 `// 错误消息`。

```suggestion
type wrapError struct {
	// message 错误消息
	message  string
	// next 原始错误
	next     error
	// pc 程序计数器，用于存储调用栈信息
	pc       uintptr
	// file 源文件名
	file     string
	// line 行号
	line     int
	// pkgPath 包路径
	pkgPath  string
	// funcName 函数名
	funcName string
}